bigquery/google/cloud/bigquery/client.py

```diff
@@ -166,6 +166,20 @@ def dataset(self, dataset_name, project=None):
 
         return DatasetReference(project, dataset_name)
 
+    def get_dataset(self, dataset_ref):
+        """API call:  refresh dataset properties via a GET request.
```

Comment:
s/refresh/read/?

Reply:
How about "Fetch the dataset referenced by ``dataset_ref``" to match the docstring for `get_job()`?

Reply:
Made change with https://github.com/GoogleCloudPlatform/google-cloud-python/pull/3973/commits/811c9b03d5825d514347e16ff43d8e1948906a2f